Intersect.Client/Core/Graphics.cs

```diff
@@ -768,6 +827,60 @@ public static void DrawFullScreenTextureFitMaximum(GameTexture tex)
             }
         }
 
+        public static void DrawFullScreenTextureAnimated(
+            string animFrameName,
+            long animFrameInterval,
+            Enum displayMode
+        )
+        {
+            sAnimFrameCount = Globals.ContentManager.GetTextureNames(TextureType.Animation)
+                .Count(s => s.Contains(animFrameName));
+
+            if (sAnimTextureTime < Timing.Global.Milliseconds)
+            {
+                sAnimTextureFrame += 1;
+                sAnimTextureTime = Timing.Global.Milliseconds + animFrameInterval;
+            }
+
+            if (sAnimTextureFrame >= sAnimFrameCount - 1)
+            {
+                sAnimTextureFrame = 0;
+            }
+
+            var animTex = Globals.ContentManager.GetTexture(
+                TextureType.Animation, animFrameName + "_" + sAnimTextureFrame + ".png"
+            );
+
+            if (animTex == null)
+            {
+                return;
+            }
+
+            switch (displayMode)
+            {
+                case DisplayModes.Default:
+                    DrawFullScreenTexture(animTex);
+
+                    break;
+                case DisplayModes.Center:
+                    DrawFullScreenTextureCentered(animTex);
+
+                    break;
+                case DisplayModes.Stretch:
+                    DrawFullScreenTextureStretched(animTex);
+
+                    break;
+                case DisplayModes.FitHeight:
+                    DrawFullScreenTextureFitHeight(animTex);
+
+                    break;
+                case DisplayModes.FitWidth:
+                    DrawFullScreenTextureFitWidth(animTex);
+
+                    break;
```

Comment:
Can you put the extra newline after the break, not before it

Reply:
got it ~